components/style/values/specified/effects.rs

```diff
@@ -172,12 +196,28 @@ impl Parse for Filter {
                 "blur" => Ok(GenericFilter::Blur((Length::parse_non_negative(context, i)?).into())),
                 "brightness" => Ok(GenericFilter::Brightness(Factor::parse(context, i)?)),
                 "contrast" => Ok(GenericFilter::Contrast(Factor::parse(context, i)?)),
-                "grayscale" => Ok(GenericFilter::Grayscale(Factor::parse(context, i)?)),
+                "grayscale" => {
+                    // Values of amount over 100% are allowed but UAs must clamp the values to 1.
+                    // https://www.w3.org/TR/filter-effects-1/#funcdef-grayscale
```

Comment:
Please never link to `TR` specs since they're usually outdated.

https://drafts.fxtf.org/filter-effects/#funcdef-filter-grayscale

Reply:
OK. Thanks!

Reply:
No problem, thanks for fixing this, and sorry for the extra hassle!